.angular-cli.json

```diff
@@ -22,6 +23,13 @@
             "styles": [
                 "styles.scss"
             ],
+            "stylePreprocessorOptions": {
+                "includePaths": [
```

Comment:
I am reading up on this includePaths, and I don't think this is being used properly.
https://github.com/angular/angular-cli/wiki/stories-global-styles

If you want these to be included in the bundle, I think these go in the "styles" section right above it. 

I could be wrong, we can discuss it